AGENTS.md

```diff
@@ -293,159 +316,42 @@ Math.PI * Math.pow(d.radius, 2)
-  - `source` - Starting point `{ lat, lng, alt }`
-  - `target` - Ending point `{ lat, lng, alt }`
-  - `arcHeight` - Height of the arc in meters
-  - `smoothHeight` - Smooth altitude transitions (default: `true`)
-  - `smoothPosition` - Smooth position transitions (default: `false`)
-  - `segmentCount` - Number of segments in the arc (default: `250`)
-  - `wrapLongitude` - Handle anti-meridian crossing (default: `true`)
-  - `tilt` - Tilt angle in degrees, -90 to 90 (default: `0`)
-  - Returns array of `[lng, lat, alt]` coordinates
-
-- **`mix(from, to, t)`** - Linear interpolation between two numbers
-- **`mixspace(start, end, mixAmount[])`** - Linear interpolation across multiple ratios
-- **`clamp(x, lower, upper)`** - Constrain a value within a range
-- **`range(stop)`** - Generate array of integers from 0 to stop-1
-- **`smoothstep(edge0, edge1, x)`** - Smooth interpolation with Hermite polynomial
-- **`segmentRatios(segmentCount, smooth)`** - Generate interpolation ratios for arcs
-- **`paraboloid(distance, sourceZ, targetZ, ratio, scaleHeight)`** - Calculate parabolic arc height
-- **`tiltPoint(point, start, end, tilt)`** - Apply tilt transformation to a point
+The `utils` object is available globally in CodeOp, AccessorOp, and ExpressionOp. It provides commonly-used functions:
+
+**Key utilities include:**
 
+- **Arc Geometry**: `getArc()` - Generate 3D arc paths between two points
+- **Color Conversion**: `hexToColor()`, `colorToHex()`, `rgbaToColor()` - Convert between color formats
+- **Geospatial**: `getDirections()` - Get routing directions between points
+- **Interpolation**: `interpolate()` - Create mapping functions between ranges
+- **Array Operations**: `cross()` - Generate all unique pairs from an array
+- **Search**: `binarySearchClosest()` - Find closest value in sorted array
+- **Distance Constants**: `FEET_TO_METERS`, `MILES_TO_METERS`, etc.
+- **Map Styles**: `CARTO_DARK`, `MAP_STYLES` - Predefined basemap URLs
+- **Random**: `mulberry32(seed)` - Deterministic pseudo-random number generator
+
+**Example usage:**
 ```javascript
-// Example: Create a 3D arc between two cities
+// Create a 3D arc between two cities
 const arc = utils.getArc({
-  source: { lat: 40.7128, lng: -74.0060, alt: 0 },    // NYC
-  target: { lat: 51.5074, lng: -0.1278, alt: 0 },     // London
+  source: { lat: 40.7128, lng: -74.0060, alt: 0 },
+  target: { lat: 51.5074, lng: -0.1278, alt: 0 },
   arcHeight: 500000,  // 500km peak height
-  tilt: 15,           // 15-degree tilt
   segmentCount: 100
 })
-```
 
-### Search and Data Utilities
-
-- **`binarySearchClosest(arr, val, i?)`** - Find the closest value in a sorted array
-  - Returns the index of the closest element
-  - Optional `i` parameter to start search from a specific index
-
-```javascript
-// Example: Find closest timestamp
-const times = [0, 100, 200, 300, 400]
-const idx = utils.binarySearchClosest(times, 250)  // Returns 2
-```
-
-### Color Utilities
-**From `utils.color`:**
```

Comment:
Removing these completely doesn't seem helpful. What about moving them to a utils API reference page in the website?